test/Unit/AESCryptoTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace NCrypt\Client;
+
+use NCrypt\Client\Library\Crypto\AESCrypto;
+use NCrypt\Client\Model\SecureNote;
+use PHPUnit\Framework\TestCase;
+
+class AESCryptoTest extends TestCase
+{
+
+    public function testReturnTypeValidatorForSecureNoteRequest()
+    {
+        $crypto = new AESCrypto();
+        $secureNoteResult = $crypto->secureNoteRequest("this is a test note");
+        $this->assertInstanceOf(SecureNote::class,$secureNoteResult);
+    }
+
+    public function testIvAndKeyLengthInSecureNoteRequest()
+    {
+        $crypto = new AESCrypto();
+        $secureNoteResult = $crypto->secureNoteRequest("this is a test note");
+
+        $lengthOfIv = strlen($secureNoteResult->getIv());
+        $this->assertEquals(16,$lengthOfIv);
```

Comment:
follow code styles, `$this->assertEquals(16,$lengthOfIv);` should be `$this->assertEquals(16, $lengthOfIv);` for example.

Reply:
it's done